### ISSUE TITLE
Experiments about cifar10

### ISSUE THREAD

**Issue by ahhnchangjun** [rvl-lab-utoronto/BFP#2]
hello 

Good to see your method, thanks

When experimenting with S-CIFAR10, may I know how you used scheduler, lr decay, milestones ?

**Comment by georgegu1997** [comment]
Thanks for your interest!

As mentioned in Section 4.1 Training details, we adopted the hyperparameters from the DER paper. The detailed hyperparameters can be found [here](https://github.com/rvl-lab-utoronto/BFP/blob/master/models/bfp/args.py#L5). 

In terms of optimizer and scheduler, please check this [file](https://github.com/rvl-lab-utoronto/BFP/blob/master/datasets/utils/continual_dataset.py#L177).
